### PR TITLE
Change Admin icon and make Dashboard filter panel color consistent

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Change Admin icon in Settings > Members > Admin
- Change Dashboard filter color with other filters
- Fix the button style in Not Found page

https://github.com/open-metadata/OpenMetadata/issues/7149

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
image

image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx [hunk 1]
```diff
@@ -183,15 +183,20 @@ const ActivityFeedList: FC<ActivityFeedListProp> = ({
               tag="button"
               variant="link"
               onClick={() => setShowThreadTypeList((visible) => !visible)}>
-              <SVGIcons alt="filter" icon={Icons.FILTERS} width="14px" />
-              <span className="tw-font-medium tw-text-grey tw-ml-1">
+              <SVGIcons
+                alt="filter"
+                className="tw-text-primary"
+                icon={Icons.FILTER_PRIMARY}
+                width="14px"
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:190]: I think we don't have any width param in SVGIcons can you double check?

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment!

**Comment by ShaileshParmar11** [comment]
make sure the admin icon should have primary colour when we hover over the admin option, work with @aniketkatkar97 if you have any doubt regards to that!

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.tsx:190]: @ShaileshParmar11  we are accepting it.

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**